internal/cli/commands/get/agent_versions.go

```diff
@@ -4,7 +4,7 @@
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//      http://www.apache.org/licenses/LICENSE-2.0
+//	http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
```suggestion
//      http://www.apache.org/licenses/LICENSE-2.0
```
I think this might be a tab on accident? It shows up as 2 chars for me in VS code